app/src/main/res/layout-land/onboarding_slide_final.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.onboarding.OnboardingSlideViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="horizontal">
+
+    <ImageView
+      android:id="@+id/slide_image_view"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      android:adjustViewBounds="true"
+      android:contentDescription="@{viewModel.contentDescription}"
+      android:scaleType="fitXY"
+      android:src="@{viewModel.slideImage}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintDimensionRatio="H, 10:9"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+      android:id="@+id/slide_title_text_view"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="8dp"
+      android:layout_marginTop="80dp"
+      android:layout_marginEnd="28dp"
+      android:fontFamily="sans-serif-medium"
+      android:gravity="center_horizontal"
+      android:text="@{viewModel.title}"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="24sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="1.0"
+      app:layout_constraintStart_toEndOf="@+id/slide_image_view"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+      android:id="@+id/slide_description_text_view"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="20dp"
+      android:layout_marginTop="16dp"
+      android:layout_marginEnd="40dp"
+      android:fontFamily="sans-serif"
+      android:gravity="center_horizontal"
+      android:text="@{viewModel.description}"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="16sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.0"
+      app:layout_constraintStart_toEndOf="@+id/slide_image_view"
+      app:layout_constraintTop_toBottomOf="@+id/slide_title_text_view" />
+
+    <Button
+      android:id="@+id/get_started_button"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
```

Comment:
The earlier comment was misleading from my side: https://github.com/oppia/oppia-android/pull/1032#discussion_r417210478

The marginStart is `28dp`

Reply:
Done